linkinator.config.json

```diff
@@ -3,6 +3,7 @@
   "skip": [
     "https://codecov.io/gh/googleapis/",
     "www.googleapis.com",
-    "setEndpoint.js"
+    "setEndpoint.js",
+    "https://www.tensorflow.org/js"
```

Comment:
I'm a little surprised we needed to add this, isn't the tensorflow.org website available?

Reply:
it is surprisingly flaky!